packages/manager/modules/constants/src/regions.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * A region is a location where the Manager is deployed
+ * A subsidiary can only be in one region
+ *
+ * Note: a subsidiary may be located in a region different from its
+ * geographical location
+ *
+ * @typedef   {object}      Region
+ * @property  {string}      name     - Name of the region
+ * @property  {string[]}    subs    - subsidiaries in this region
+ */
+export default {
+  CA: {
+    name: 'CA',
+    subs: [
+      'ASIA',
+      'AU',
+      'CA',
+      'QC',
+      'SG',
+      'WE',
+      'WS',
+    ],
+  },
+  EU: {
+    name: 'EU',
```

Comment:
It's easier to handle than the object keys:

```js
map(regions, 'name') => ['EU', 'CA', 'US']
```

instead of

```js
map(toPairs(a), ([key, value]) => key) => ['EU', 'CA', 'US']
```